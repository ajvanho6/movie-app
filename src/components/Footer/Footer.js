import React, {Component} from 'react';

import './Footer.scss';

class Footer extends Component {
    state = {};

    render() {
        return (
            <div className="m-app-footer">
                Crafted by Ivan Baresic with <span role="img" aria-label="heart">&#128420;</span>
            </div>
        );
    }
}

export default Footer;
