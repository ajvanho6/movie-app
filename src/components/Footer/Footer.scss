@import '../../styles/config';
@import '../../styles/mixins';

$component-class: m-app-footer;

.#{$component-class} {
    height: 40px;
    display: flex;
    align-items: center;
    justify-content: center;
    width: 100%;
    background: map-get($colors, background-11);

    span {
        margin-left: 5px;
    }
}
