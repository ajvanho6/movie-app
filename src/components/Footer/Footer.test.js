import React from 'react';
import {shallow} from 'enzyme';
import Footer from './Footer';

describe('Footer component', () => {
    it('Footer should render correctly', () => {
        const component = shallow(<Footer />);
        expect(component).toMatchSnapshot();
    });
});
