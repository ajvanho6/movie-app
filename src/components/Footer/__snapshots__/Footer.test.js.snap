// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer should render correctly 1`] = `ShallowWrapper {}`;
